app/controllers/admin/api/cms/sections_controller.rb

```diff
@@ -2,7 +2,8 @@ class Admin::Api::CMS::SectionsController < Admin::Api::CMS::BaseController
   ##~ sapi = source2swagger.namespace("CMS API")
   ##~ @parameter_section_id = { :name => "id", :description => "ID of the section", :dataType => "int", :required => true, :paramType => "path" }
 
-  wrap_parameters :section, include: [:title, :public, :parent_id, :partial_path]
+  ALLOWED_PARAMS = %i[parent_id title system_name public partial_path].freeze
```

Comment:
Just curious - why the order is like this? 
I mean, it's different compared to the original list in `wrap_parameters`.

Maybe it makes sense to order them alphabetically?